src/styles/watch.scss

```diff
@@ -0,0 +1,75 @@
+.stopwatch {
+  height: $size * 2;
+  width: $size * 2;
+  margin: 0;
+  position: absolute;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  border-radius: 50%;
+  border: 30px groove black;
+  animation: 10s linear infinite none bgcolors;
+  background-color: #aaa;
+
+  &__middle {
+    width: $size * .2;
+    height: $size * .2;
+    position: absolute;
+    top: $size - ($size * .2) / 2;
+    border-radius: 50%;
+    background-color: darkslateblue;
+    z-index: 3;
+    opacity: 0.6;
+  }
+
+  &__big-pointer {
+    width: 1px;
+    height: $size;
+    position: absolute;
+    background-color: red;
+    transform-origin: left bottom;
+    animation: 5s linear infinite none rotation;
+  }
```

Comment:
Both the hands and the center have wrong sizes and colors [Reference](https://github.com/mate-academy/layout_stop-watch#:~:text=Use%20reference%20image%20below%20to%20create%20a%20stopwatch)